core/src/main/java/org/apache/carbondata/core/view/MVProvider.java

```diff
@@ -121,70 +100,63 @@ private SchemaProvider getSchemaProvider(String databaseName) {
     return schemaProvider;
   }
 
-  public MVSchema getSchema(MVManager viewManager,
-                            String databaseName, String viewName) throws IOException {
-    SchemaProvider schemaProvider = this.getSchemaProvider(databaseName);
+  public MVSchema getSchema(MVManager viewManager, String databaseName, String viewName)
+      throws IOException {
+    SchemaProvider schemaProvider = this.getSchemaProvider(viewManager, databaseName);
     if (schemaProvider == null) {
       return null;
     }
     return schemaProvider.retrieveSchema(viewManager, viewName);
   }
 
-  List<MVSchema> getSchemas(MVManager viewManager,
-                            String databaseName, CarbonTable carbonTable) throws IOException {
-    SchemaProvider schemaProvider = this.getSchemaProvider(databaseName);
+  List<MVSchema> getSchemas(MVManager viewManager, String databaseName, CarbonTable carbonTable)
+      throws IOException {
+    SchemaProvider schemaProvider = this.getSchemaProvider(viewManager, databaseName);
     if (schemaProvider == null) {
       return Collections.emptyList();
     } else {
       return schemaProvider.retrieveSchemas(viewManager, carbonTable);
     }
   }
 
-  List<MVSchema> getSchemas(MVManager viewManager,
-                            String databaseName) throws IOException {
-    SchemaProvider schemaProvider = this.getSchemaProvider(databaseName);
+  List<MVSchema> getSchemas(MVManager viewManager, String databaseName) throws IOException {
+    SchemaProvider schemaProvider = this.getSchemaProvider(viewManager, databaseName);
     if (schemaProvider == null) {
       return Collections.emptyList();
     } else {
       return schemaProvider.retrieveAllSchemas(viewManager);
     }
   }
 
-  void saveSchema(MVManager viewManager, String databaseName,
-                  MVSchema viewSchema) throws IOException {
-    SchemaProvider schemaProvider = this.getSchemaProvider(databaseName);
+  void saveSchema(MVManager viewManager, String databaseName, MVSchema viewSchema)
```

Comment:
please keep the style as before, revert the method signature changes, just change `SchemaProvider schemaProvider = this.getSchemaProvider(viewManager, databaseName);`

Reply:
OK